### ISSUE TITLE
Error loading .gfa file in BandageNG - "Malformed path string for path 'EU305676.1'"

### ISSUE THREAD

**Issue by florenmartino** [asl/BandageNG#164]
Hi, I'm encountering an error when attempting to open a .gfa file in BandageNG. The following message appears:

There was an error when attempting to load
/home/florenmartino/Documents/Code/Dylans_results/AS5/AS5_24mo.all_contigs.minimus.multi_assembly.norm.gfa:
malformed path string for path 'EU305676.1', cannot reconstruct path through the graph

I've verified with others gfa files, and I could open the same file in another computer using BandageNG.
I'm using BandageNG in a computer under

Linux system76-pc 6.9.3-76060903-generic #202405300957~1726766035~22.04~4092a0e~dev-Ubuntu SMP PREEMPT_DY x86_64 x86_64 x86_64 GNU/Linux

The Bandage that Im using is BandageNG-9eb84c2-x86_64.AppImage from

https://github.com/asl/BandageNG/releases/tag/continuous

Could you please confirm if this file format is supported or if there might be an issue with specific path strings in the .gfa file?

Thanks for your help!

**Comment by asl** [comment]
Will you please attach the graph in question?

**Comment by florenmartino** [comment]
I try in a Mac, and same issue with the file. 
I was able to open it using the standard Bandage software (Ryan Wick, the older version), and it worked.

[AS5_24mo.all_contigs.minimus.clean.multi_assembly.norm.txt](https://github.com/user-attachments/files/17588887/AS5_24mo.all_contigs.minimus.clean.multi_assembly.norm.txt)

I uploaded as .txt but of course im openning with .gfa

Thanks in advance!
